com.unity.ml-agents/Runtime/ActionMasker.cs

```diff
@@ -118,7 +118,7 @@ internal bool[] GetMask()
         void AssertMask()
         {
             // Action Masks can only be used in Discrete Control.
-            if (m_BrainParameters.vectorActionSpaceType != SpaceType.Discrete)
+            if (m_BrainParameters.vectorActionSpaceType != ActionType.Discrete)
```

Comment:
`ActionSpaceType`? That's what we're calling it in BrainParameters.

Reply:
ActionSpaceType sounds weird to me. I think Action Type makes more sense to me as I am not sure where space comes from.
@andrewcoh

Reply:
People typically use space to refer to the set of actions an agent has i.e. continuous/discrete action spaces.  Also, gym organizes their action spaces in a directory 'spaces' https://github.com/openai/gym/tree/master/gym/spaces

You could argue that action space type refers to the nature of the space i.e. discrete/continuous but I'm not sure it's worth arguing

Reply:
ok, I guess you can parse `m_BrainParameters.vectorActionSpaceType` as (vector action) (space type), so "SpaceType" would be reasonable.